src/libnnstreamer-edge/nnstreamer-edge-util.h

```diff
@@ -41,6 +42,13 @@ extern "C" {
 #define FALSE (0)
 #endif
 
+/**
+ * @brief The max size to allocate memory. This would be defined in C99 limits.h.
+ */
+#ifndef SIZE_MAX
+#define SIZE_MAX ((size_t) -1)
```

Comment:
`nns_size_t` ?

Reply:
No, arg of malloc() is size_t and its type may be uint32 but nns_size_t is uint64.

Reply:
You're right, thanks